protocol/contracts/lpp/src/state/total.rs

```diff
@@ -81,14 +81,24 @@ where
     ) -> Result<&Self, ContractError> {
         self.total_interest_due = self.total_interest_due_by_now(&ctime);
 
+        let updated_total_principal_due =
```

Comment:
the naming is not correct - the amount is not yet "updated"
replacing it just with "new_total" would tell us more